images/deployer/Dockerfile

```diff
@@ -9,4 +9,11 @@
 #
 FROM openshift/origin
 
+RUN yum install -y nfs-utils sudo
+
+# Edit sudoers file
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
at this point, we should probably have a separate recycler image... this doesn't seem good to do to our stock deployer image